diff --git a/dist/backends/single_buffer.js b/dist/backends/single_buffer.js
index 234af8287e8b97da89d139644c95999b16d1d789..75bf8a2c0ea03ac717c4002595d377c6bf65ba0d 100644
--- a/dist/backends/single_buffer.js
+++ b/dist/backends/single_buffer.js
@@ -46,7 +46,7 @@ var __addDisposableResource = (this && this.__addDisposableResource) || function
             if (async) inner = dispose;
         }
         if (typeof dispose !== "function") throw new TypeError("Object not disposable.");
-        if (inner) dispose = function() { try { inner.call(this); } catch (e) { return Promise.reject(e); } };
+        if (inner) dispose = function () { try { inner.call(this); } catch (e) { return Promise.reject(e); } };
         env.stack.push({ value: value, dispose: dispose, async: async });
     }
     else if (async) {
@@ -67,7 +67,7 @@ var __disposeResources = (this && this.__disposeResources) || (function (Suppres
                     if (!r.async && s === 1) return s = 0, env.stack.push(r), Promise.resolve().then(next);
                     if (r.dispose) {
                         var result = r.dispose.call(r.value);
-                        if (r.async) return s |= 2, Promise.resolve(result).then(next, function(e) { fail(e); return next(); });
+                        if (r.async) return s |= 2, Promise.resolve(result).then(next, function (e) { fail(e); return next(); });
                     }
                     else s |= 1;
                 }
@@ -710,7 +710,7 @@ export class SingleBufferStore extends BufferView {
         return this._fs;
     }
     set fs(fs) {
-        if (this.buffer instanceof SharedArrayBuffer)
+        if (typeof SharedArrayBuffer !== 'undefined' && this.buffer instanceof SharedArrayBuffer)
             fs === null || fs === void 0 ? void 0 : fs.attributes.set('no_id_tables', true);
         this._fs = fs;
     }
